package.json

```diff
@@ -3,12 +3,16 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    "dev": "npx nx run-many --target=dev --projects=server,client"
+    "dev": "npx nx run-many --target=dev --projects=server,client",
+    "prepare": "husky install"
   },
   "private": true,
   "devDependencies": {
     "@nx/js": "16.8.1",
-    "nx": "16.8.1"
+    "husky": "^8.0.0",
+    "lint-staged": "^14.0.1",
+    "nx": "^16.8.1",
+    "tsc-files": "^1.1.4"
```

Comment:
@celikmurat1845  and @metehancelik I thinks this package is only used by the client, so better to add to the client.
Maybe you are confused because lintstaged and husky are at the top. But keep in mind that they are required at the top level to interact with .git file, and this is not the case for your little, shiny tsc-files :)

Reply:
done @demirtasdurmus